requirements/base.in

```diff
@@ -12,7 +12,7 @@ django-storages
 django-structlog
 fiona --no-binary fiona
 geopandas
-graphene-django
+graphene-django==3.1.5
```

Comment:
question: do you know how pinning the version here will interact with dependabot? or will we just find out on monday?

Reply:
We already have this particular dependency as a PR: https://github.com/techmatters/terraso-backend/pull/1073

I am not sure what dependabot will do for 3.2.x/3.3.